velox/docs/functions/spark/datetime.rst

```diff
@@ -113,7 +113,11 @@ These functions support TIMESTAMP and DATE input types.
     The format follows Spark's
     `Datetime patterns
     <https://spark.apache.org/docs/latest/sql-ref-datetime-pattern.html>`_.
-    Returns NULL for parsing error or NULL input. Throws exception for invalid format. ::
+    Using `Simple <https://docs.oracle.com/javase/8/docs/api/java/text/SimpleDateFormat.html>`
```

Comment:
This link is not properly hidden beneath the text. Would you like to fix it?

Reply:
Sure. Fixed.